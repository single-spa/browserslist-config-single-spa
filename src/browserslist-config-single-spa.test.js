import browserslist from "browserslist";
import queries from "./browserslist-config-single-spa.js";

describe("browsereslist-config-single-spa", () => {
  it("exports a valid browserslist config for single-spa", () => {
    const browsers = browserslist(queries);
    expect(browsers).toMatchSnapshot();
  });
});
