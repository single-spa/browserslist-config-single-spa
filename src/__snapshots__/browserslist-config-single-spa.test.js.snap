// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`browsereslist-config-single-spa exports a valid browserslist config for single-spa 1`] = `
Array [
  "chrome 93",
  "chrome 92",
  "chrome 91",
  "chrome 90",
  "chrome 89",
  "chrome 88",
  "chrome 87",
  "chrome 86",
  "chrome 85",
  "chrome 84",
  "edge 93",
  "edge 92",
  "edge 91",
  "edge 90",
  "firefox 92",
  "firefox 91",
  "firefox 90",
  "firefox 89",
  "firefox 88",
  "firefox 87",
  "firefox 86",
  "firefox 85",
  "firefox 84",
  "firefox 83",
  "ie 11",
  "safari 14.1",
  "safari 14",
  "safari 13.1",
  "safari 13",
  "safari 12.1",
  "safari 12",
  "safari 11.1",
  "safari 11",
]
`;
